### PR TITLE
Csv pipe

### PR DESCRIPTION
**Pull request by AndreasArvidsson**
Support pipe operator in csv
`remove|chuck`

### CODE REVIEW

File: src/csv_overrides.py [hunk 1]
```diff
@@ -274,7 +274,8 @@ def read_file(
             csv_error(path, i, "Duplicate identifier", value)
             continue
 
-        result[key] = value
+        for k in key.split("|"):
+            result[k.strip()] = value
```
Comment (pokey) [inline src/csv_overrides.py:278]: I'm a bit surprised that this code works.  I would think that in https://github.com/pokey/cursorless-talon/blob/e75cda037cca8a2de734027446234893f1661143/src/csv_overrides.py#L129 we would just end up with whichever key appears last.  Presumably you've checked that this code works?  Any idea why it doesn't get overridden in that line?

Man we really need tests for this stuff 😅

Comment (AndreasArvidsson) [reply src/csv_overrides.py:278]: You're probably right I need to test some more.

Comment (AndreasArvidsson) [reply src/csv_overrides.py:278]: Now it's testing and working

File: src/csv_overrides.py [hunk 2]
```diff
@@ -136,16 +136,15 @@ def update_dicts(
                     "value": value,
                     "list": default_list_name,
                 }
-            else:
-                raise
```
Comment (pokey) [inline src/csv_overrides.py:140]: Should we at least log something to terminal here?  I don't think it's great to just fail silently

**Review by pokey — COMMENTED** [review]
Yeah this looks more likely to work 😊.  See minor comment

File: src/csv_overrides.py (hunk 2 above)
Comment (AndreasArvidsson) [reply src/csv_overrides.py:140]: It doesn't fail; it just skips that one. But we can add a comment

Comment (AndreasArvidsson) [reply src/csv_overrides.py:140]: done

File: src/csv_overrides.py [hunk 3]
```diff
@@ -137,15 +138,20 @@ def update_dicts(
                     "list": default_list_name,
                 }
             else:
-                raise
+                print(f"ERROR: Unknown identifier '{value}'")
+                has_errors = True
+
+    if has_errors:
+        app.notify("Cursorless settings error; see log")
```
[the remark this answers is not in the document]
Comment (AndreasArvidsson) [reply src/csv_overrides.py:145]: No I'm not sure if it's ever have actually raised anything. Okay I will restore it

[the remark this answers is not in the document]
Comment (AndreasArvidsson) [reply src/csv_overrides.py]: done

**Review by pokey — APPROVED** [review]

**Pull request merged**